python/cugraph-pyg/cugraph_pyg/data/cugraph_store.py

```diff
@@ -19,33 +19,37 @@
 from itertools import chain
 from functools import cached_property
 
-import warnings
-
+# numpy is always available
 import numpy as np
-import cudf
+import pandas
 
-from cugraph.utilities.utils import import_optional, MissingModule
 
-# FIXME remove these imports and replace PG with FeatureStore
-from cugraph.experimental import MGPropertyGraph
+import cugraph
+
+from cugraph.utilities.utils import import_optional, MissingModule
+import cudf
 
 # FIXME drop cupy support and make torch the only backend (#2995)
-cupy = import_optional("cupy")
+import cupy
+
+import dask.dataframe as dd
+from dask.distributed import get_client
+
+
 torch = import_optional("torch")
-cugraph = import_optional("cugraph")
-cugraph_service_client = import_optional("cugraph_service_client")
 
 Tensor = None if isinstance(torch, MissingModule) else torch.Tensor
 NdArray = None if isinstance(cupy, MissingModule) else cupy.ndarray
 
 TensorType = Union[Tensor, NdArray]
-CuGraphGraph = None if isinstance(cugraph, MissingModule) else cugraph.MultiGraph
-CGSGraph = (
-    None
-    if isinstance(cugraph_service_client, MissingModule)
-    else cugraph_service_client.RemoteGraph
-)
-StructuralGraphType = Union[CuGraphGraph, CGSGraph]
+
+
+def torch_as_array(a):
+    if isinstance(a, cupy.ndarray):
+        t = torch.from_dlpack(a)
+    elif isinstance(a, cudf.Series):
+        t = torch.from_dlpack(a.to_dlpack())
+    return t
```

Comment:
Motivation of this fix . Should have been clearer, this gives you managed memory for free with Pytorch as dlpack  fails .   

```python
import rmm      
import torch
import cudf
rmm.reinitialize(managed_memory= True)
torch.cuda.memory.change_current_allocator(rmm.rmm_torch_allocator)
```
Below works:
```python
ar = cudf.Series([1]).repeat(1_000_000_000).values
t1 = torch.as_tensor(ar, device='cuda')
t1
tensor([1, 1, 1,  ..., 1, 1, 1], device='cuda:0')
```
Below Fails: 
```python
ar = cudf.Series([1]).repeat(1_000_000_000).values
t2 = torch.from_dlpack(ar.toDlpack())
```
```python
RuntimeError: Unsupported device_type: 13
```

Reply:
I thought I changed this

Reply:
@VibhuJawa the push failed, please review again.

Reply:
Thanks. Approved it .